testing/btest/scripts/base/frameworks/storage/redis-sync.zeek

```diff
@@ -30,27 +30,31 @@ event zeek_init()
 	local key = "key1234";
 	local value = "value1234";
 
-	local open_res = Storage::Sync::open_backend(Storage::REDIS, opts, string, string);
-	print "open_result", open_res;
+	local res = Storage::Sync::open_backend(Storage::REDIS, opts, string, string);
+	print "open_result", res;
 
-	local b = open_res$value;
+	local b = res$value;
 
-	local res = Storage::Sync::put(b, [ $key=key, $value=value ]);
+	res = Storage::Sync::put(b, [ $key=key, $value=value ]);
 	print "put result", res;
 
-	local res2 = Storage::Sync::get(b, key);
-	print "get result", res2;
-	if ( res2$code == Storage::SUCCESS && res2?$value )
-		print "get result same as inserted", value == ( res2$value as string );
+	res = Storage::Sync::get(b, key);
+	print "get result", res;
+	if ( res$code == Storage::SUCCESS && res?$value )
+		print "get result same as inserted", value == ( res$value as string );
 
 	local value2 = "value5678";
 	res = Storage::Sync::put(b, [ $key=key, $value=value2, $overwrite=T ]);
 	print "overwrite put result", res;
 
-	res2 = Storage::Sync::get(b, key);
-	print "get result", res2;
-	if ( res2$code == Storage::SUCCESS && res2?$value )
-		print "get result same as inserted", value2 == ( res2$value as string );
+	res = Storage::Sync::get(b, key);
+	print "get result", res;
+	if ( res$code == Storage::SUCCESS && res?$value )
+		print "get result same as inserted", value2 == ( res$value as string );
+
+	# Validate that $overwrite=F returns KEY_EXISTS
+	res = Storage::Sync::put(b, [ $key=key, $value=value2, $overwrite=F ]);
+	print "no overwrite put result", res;
 
```

Comment:
Could make sense to do another `get()` and ensure the value really wasn't overwritte (but would need something else but `$value=value2` as that just writes the same value again.).